### PR TITLE
Simplify Readme

### PR DESCRIPTION
**Pull request by pierrelalanne**
In this PR, we simplify `ggshield`'s README.md and we point to the [public documentation](https://docs.gitguardian.com/ggshield-docs/getting-started) instead when applicable.  

We keep the following sections in the README :
- How to install
- How to authenticate and start using the project
- A link to the doc
- Related open source project
- License

Later, we may want to include a small demo of the tool. Also, we may want to get rid of the output section ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +67,7 @@ ggshield supports **Python 3.7 and newer**.
 
 The package should run on MacOS, Linux and Windows.
 
-#### Updating
+### Updating
 
 To update ggshield you can add the option `-U/--upgrade` to the pip install command.
```
Comment (agateau-gg) [inline README.md:72]:
```suggestion
To update ggshield you can add the option `-U/--upgrade` to the `pip install` command.
```

**Review by agateau-gg — CHANGES_REQUESTED** [review]
I think we need to wet the appetite of the reader a bit more.

(Disclaimer: I am not a marketing person, but I just pretend to be one in pull-request reviews!)

I suggest replacing the "Initial setup" and "Documentation" sections with this:

---
## Initial setup

To use `ggshield` you need to authenticate against GitGuardian servers. To do so, use the `ggshield auth login` command. This command automates the provisioning of a personal access token and its configuration on the local workstation.

You can learn more about it from [`ggshield auth login` documentation](https://docs.gitguardian.com/internal-repositories-monitoring/ggshield/reference/auth/login).

Alternatively, you can create your personal access token manually and you can store it in the `GITGUARDIAN_API_KEY` environment variable to complete the setup.

## Getting started

You can now use `ggshield` to search for secrets:

- in files: `ggshield secret scan path -r .`
- in repositories: `ggshield secret scan repo .`
- in Docker images: `ggshield secret scan docker ubuntu:22.04`
- in Pypi packages: `ggshield secret scan pypi flask`
- and more, have a look at `ggshield secret scan --help` output for details.

You can also search for issues in your IaC code:

```
ggshield iac scan .
```

## Integrations

You can integrate `ggshield` in your [CI/CD workflow](https://docs.gitguardian.com/ggshield-docs/integrations/overview#cicd-integrations-secrets-detection-in-your-cicd-workflow).

To catch errors earlier, use `ggshield` as a [pre-commit, pre-push or pre-receive Git hook](https://docs.gitguardian.com/ggshield-docs/integrations/overview#git-hooks-prevent-secrets-from-reaching-your-vcs).

## Learn more

For more information, have a look at [the documentation](https://docs.gitguardian.com/ggshield-docs/getting-started)

**Comment by pierrelalanne** [comment]
I fully agree with your comment, I integrated your suggestion.  
I am still not convinced by the output section, but I think we should keep it until we include a nice demo of the tool in the README.

**Comment by agateau-gg** [comment]
> I fully agree with your comment, I integrated your suggestion. I am still not convinced by the output section, but I think we should keep it until we include a nice demo of the tool in the README.

I agree. Let's merge this: it solves the issue of duplicated information. We can iterate on the README and add a demo later.

**Pull request merged**